### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by kolya2731**

### CODE REVIEW

File: app/battle/fight.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def fight(name_1: dict, name_2: dict) -> None:
```
Comment (vasylhnatiuk) [inline app/battle/fight.py:1]: don't create separate file for one small function

File: app/battle/check_hp.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+def check_hp(name: dict) -> None:
```
Comment (vasylhnatiuk) [inline app/battle/check_hp.py:1]: don't create separate file for one small function

File: app/main.py [hunk 3]
```diff
@@ -1,206 +1,39 @@
...
-    # apply armour
-    mordred["protection"] = 0
-    for a in mordred["armour"]:
-        mordred["protection"] += a["protection"]
+    arthur = knights_config["arthur"]
+    apply_all(arthur)
 
-    # apply weapon
-    mordred["power"] += mordred["weapon"]["power"]
+    mordred = knights_config["mordred"]
```
Comment (vasylhnatiuk) [inline app/main.py:15]: don't repeat code, use loops

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
Create class Knight and save all of knight data in class as atributes, also save functions as methods of this class

**Review by vasylhnatiuk — CHANGES_REQUESTED** [review]
.

**Review by pavlejviki — CHANGES_REQUESTED** [review]

**Review by Polyakiv-Andrey — APPROVED** [review]
Good job!

**Pull request closed without merging**